types/esbuild-copy-static-files/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for esbuild-copy-static-files 0.1
+// Project: https://github.com/nickjj/esbuild-copy-static-files
+// Definitions by: David Enke <https://github.com/davidenke>
+//                 Tom Kopp <https://github.com/TomKopp>
```

Comment:
you would have this person to explicitly give OK to use this handle as the owner of the DT module, due to send notifications, etc.
Otherwise, just give a credit